gp-populate-anything/gppa-merge-tag-context-modifier.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Gravity Perks // Populate Anything // Replace Merge Tags in Specific Context
+ * https://gravitywiz.com/documentation/gravity-forms-populate-anything/
+ *
+ * @video https://www.loom.com/share/d626a60769ee48579b6f426a677918a3
+ *
+ * This snippet allows you to replace a merge tag in the context of an entry selected in a GPPA-populated field. For
+ * example, if you populate a Drop Down field with entries and then have an HTML field which includes the {all_fields}
+ * merge tag, you could use the context modifier to specify that Drop Down field as the context like so:
+ *
+ * {all_fields:context[1]}
+ *
+ * In this example, we'll assume that the ID of the Drop Down field is 1.
+ *
+ * Note: This currently is limited to entries.
```

Comment:
Let's update this to:
>  Note: This snippet is currently limited to entries.

I wouldn't typically have you update something this minor but since we're making another change already, I figured it wouldn't hurt. 😅